automatic/vim/README.md

```diff
@@ -0,0 +1,19 @@
+# <img src="https://raw.githubusercontent.com/chocolatey-community/chocolatey-coreteampackages/master/icons/vim.svg" width="48" height="48"/> [vim](https://chocolatey.org/packages/vim)
+
+Vim is a highly configurable text editor built to enable efficient text editing. It is an improved version of the vi editor distributed with most UNIX systems.
+
+Vim is often called a programmer’s editor, and so useful for programming that many consider it an entire IDE. It’s not just for programmers, though. Vim is perfect for all kinds of text editing, from composing email to editing configuration files.
+
+This package provides official nightly build.
+
+The software is installed to chocolatey tools directory.
+
+#### Parameters
```

Comment:
Its named `Package parameters`. You miss `Features` too.

Also, I suggest in notes to specify relation to other packages and why this one differes. (like the one you gave on gallery).